raiden/network/transport/matrix/client.py

```diff
@@ -241,6 +241,12 @@ def hangup(self, room_id: RoomID, call_id: str) -> None:
         message = {"type": RTCMessageType.HANGUP.value, "call_id": call_id}
         self._send_signalling(room_id, message)
 
+    def send_to_device_message(self, user_ids: Iterable[str], text: str) -> None:
+        # TODO: send just to active user id
```

Comment:
FYI there's already code to filter for online peers: https://github.com/raiden-network/raiden/blob/9ddcfea09f36aea9346e8b22926fe35edfdad27b/raiden/network/transport/matrix/transport.py#L1369 (when passing `require_online_peer=True`).

Reply:
Cool, thanks! Should we do this here or just send to everyone?

Reply:
I think it'd be better to add this code there on `transport.py`, filter online users, and compose the message's payload there. This format is more or less specific to Raiden, and since this function does so little, we can put it there.